src/defined-types/journal.type.ts

```diff
@@ -0,0 +1,25 @@
+enum EntryType{
+    SHORT = 'short',
+    LONG = 'long'
+}
+
+enum EntryMood{
+    VERY_HAPPY = 'veryHappy',
+    HAPPY = 'happy',
+    NEUTRAL = 'neutral',
+    SAD = 'sad',
+    VERY_SAD = 'verySad',
+}
+
+export interface JournalEntry {
+    _id?: string,
+    userId: string,
+    templateId?: string,
+    createdDate: Date, 
+    lastEditedDate: Date,
+    title: string,
+    text: number,
+    photo?: number[],
```

Comment:
Let's change `photo?: number[]` to `photos: number[][]` since one entry can have more than one photo.  The type `number[]` indicates that a photo can be a binary array. The type `number[][]` indicates that photos should be an array of binary arrays.